server/neptune/workflows/internal/prrevision/github/activity_executor.go

```diff
@@ -0,0 +1,30 @@
+package github
+
+import (
+	"context"
+
+	"github.com/runatlantis/atlantis/server/neptune/workflows/activities"
+	"go.temporal.io/sdk/workflow"
+)
+
+type githubActivities interface {
+	GithubListPRs(ctx context.Context, request activities.ListPRsRequest) (activities.ListPRsResponse, error)
+	GithubListModifiedFiles(ctx context.Context, request activities.ListModifiedFilesRequest) (activities.ListModifiedFilesResponse, error)
+}
+
+// abstracts configuring the TQ for testing
+type ActivityExecutor struct {
```

Comment:
Doesn't feel like a relevant abstraction especially with the name and I find it hard to see how this would make testing easier because you still have to test which task queue was passed in.  I think if you had an abstraction for fetching all PRs and then getting all the modified files and then turning those into roots, that would be better.

Reply:
As discussed async, I've removed this abstraction since I can assert the TQ names in the mock activity invocations. Also, choosing to leave the current structure as is to avoid complexity with processing slow and old PRs simultaneously.